x-pack/legacy/plugins/monitoring/public/components/cluster/overview/elasticsearch_panel.js

```diff
@@ -124,7 +124,7 @@ function renderLog(log) {
     <EuiFlexGroup wrap responsive={false} gutterSize="xs">
       {log.levels.map((level, index) => (
         <EuiFlexItem grow={false} key={index}>
-          <EuiToolTip position="top" content={logLevelText[level.level]}>
+          <EuiToolTip position="top" content={logLevelText[level.level] || logLevelText.info}>
```

Comment:
Should we also do `Unknown` here too?

Reply:
It'll use a predefined message here: https://github.com/elastic/kibana/pull/53318/files#diff-2656fbaac4e2fcb16a5b771c47beb283R106, which I think is already kind of like a default/fallback

Or, should I maybe add an `unknown` value to `logLevelText`?

Reply:
I'd do `unknown`